flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -38,9 +38,6 @@ class FlutterLocalNotificationsPlugin {
     } else if (defaultTargetPlatform == TargetPlatform.macOS) {
       FlutterLocalNotificationsPlatform.instance =
           MacOSFlutterLocalNotificationsPlugin();
-    } else if (defaultTargetPlatform == TargetPlatform.linux) {
```

Comment:
Hi @proninyaroslav. Wanted to see if I could get your help to confirm something in case I missed anything. With this PR and the change done in https://github.com/MaikuB/flutter_local_notifications/pull/2371, the stub implementation in the Linux package can be removed. Is this correct?

Reply:
Well, if the plugin registration occurs in a separate package with its own method, then registration here is already redundant.

Reply:
Thanks